src/other/Table.js

```diff
@@ -307,55 +305,67 @@
             .append("td")
         ;
         cells[this.renderHtmlDataCells() ? "html" : "text"](function (d, idx) { 
-            var retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            var retVal;
+            if (!(d instanceof Widget)) {
+                retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            }
             return retVal; 
         });
         cells.exit()
             .remove()
-        ;      
+        ;
+
         rows.each(function(tr,trIdx){
-            d3.select(this).selectAll("td").each(function(tdContents,tdIdx){
+            var dis = d3.select(this);
+            dis.selectAll("td").each(function(tdContents,tdIdx){    
                 var alignment = context.getColumnAlignment(tdContents);
-                d3.select(this)
+                var el = d3.select(this);
+                el
+                    .style({
+                        "height": null,
+                        "text-align": alignment
+                    })
                     .classed("tr-"+trIdx+"-td-"+tdIdx,true)
-                    .style("text-align", alignment)
                 ;
+                if (tdContents instanceof Widget) {
+                    if (context.pagination()) {
+                        console.log("Warning: displaying another widget in the table may cause problems with pagination");
+                    }
+                    el.style({
+                        "height": context.minWidgetHeight() + "px",
+                        "width": context.minWidgetWidth() + "px"
+                    });
+                    console.log(tdContents.classID());
+                    console.log("this.offsetHeight:", this.offsetHeight);
+                    console.log('el.style("height"): ', el.style("height"));
+                    console.log('el.node().offsetHeight: ', el.node().offsetHeight);
+                    tdContents.target(null);
+                    tdContents.target(this);
+                    tdContents._parentWidget = context;
+                    if (tdContents._class.indexOf("childWidget") < 0) {
+                        tdContents._class = "childWidget " + tdContents._class;
+                    }
+                    tdContents.render();
+                    console.log("Widget._size.height: ", tdContents._size.height);
+                    console.log(" ")
```

Comment:
@GordonSmith referring to the IE problems...   I'm not sure what is happening there...  I added the above console logs to debug.  Can you look at them in Chrome and IE? 
http://rawgit.com/dtsnell4/Visualization/GH-1161_Table_Child_Widgets/demos/dermatology.html?src/other/Table.widget
In Chrome all of those evaluate to the proper height (200), however in IE the `el.style("height")` evaluates to 0 (and thus the `Widget._size.height`).  This must be why the widgets have no height in the DOM, but I have no idea why or what to do about it...  

Again this is just an issue with the widgets, regular tables are unaffected.